app/core/services/image_manipulation/svg_utils.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+import io
+import cairosvg
+
+
+def svg_to_png(svg_bytes_io: io.BytesIO) -> io.BytesIO:
+    png_bytes_io = io.BytesIO()
+    cairosvg.svg2png(bytestring=svg_bytes_io.getvalue(), write_to=png_bytes_io)
+    png_bytes_io.seek(0)
```

Comment:
I think it is better to close the BytesIO in input since it is replaced by the png one